.gitmodules

```diff
@@ -1,3 +0,0 @@
-[submodule "docker-images"]
-	path = docker-images
-	url = https://github.com/fetchai/docker-images.git
```

Comment:
Doesn't seem like we need this submodule. Removing should simply the workflows and cloning.